dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/logging/source/AccessLogSource.java

```diff
@@ -344,13 +349,16 @@ private static AccessLogDataFormatter populateCustomFormatters(Map<String, HashS
                     case "%C": builder.add(addCookiesField           (format)); break;
                     case "%D": builder.add(addRequestElapsedTimeField(format)); break;
                     case "%i":
-                        if (fields.get("%i").contains("User-Agent")) {
+                        // Error message was printed earlier in populateSetters(), so we just break in this case
+                        if (fields.get("%i") == null) {
+                            break;
+                        }
+                        if (fields.get("%i").contains(USER_AGENT_HEADER)) {
```

Comment:
does this check need to ignore case?

Reply:
nope, the method that lowercases the header names is called before this method (and the USER_AGENT_HEADER var is all lowercase)